### ISSUE TITLE
AroundTheWorld inheritance

### ISSUE THREAD

**Issue by garside** [Freshly/spicerack#16]
```ruby
class BaseFoo
  include AroundTheWorld
  
  def perform
    raise NotImplementedError
  end
  around_method :perform do
    super
    "😄"
  end
end

class SomeFoo < BaseFoo
  def perform
    "😢"
  end
end
```

*problem:*

```ruby
expect(SomeFoo.new.perform).to eq "😄"
# => fails; got "😢"
```

**Comment by garside** [comment]
did I do something wrong? Is this a black magic inheritance hill to far?

**Comment by arettberg** [comment]
this was solved by #18, then we decided it's a bad idea 💀
